javascript/commons/Countdown.js

```diff
@@ -98,52 +98,79 @@ liquipedia.countdown = {
 	},
 	setCountdownString: function ( timerObjectNode ) {
 		const countdownElem = timerObjectNode.querySelector( '.timer-object-countdown' );
-		let datestr = '', live = 'LIVE';
+		let datestr = '';
 
-		if ( typeof timerObjectNode.dataset.countdownEndText !== 'undefined' ) {
-			live = timerObjectNode.dataset.countdownEndText;
-		}
 		if ( timerObjectNode.dataset.timestamp !== 'error' ) {
-			const differenceInSeconds = Math.floor(
-				parseInt( timerObjectNode.dataset.timestamp ) - ( Date.now().valueOf() / 1000 )
-			);
+			const countdownContent = this.getCountdownStringAndClassName( timerObjectNode, countdownElem );
```

Comment:
Alternative without changing the return values in `getCountdownStringAndClassName`  would be

```suggestion
			let {text, className} = this.getCountdownStringAndClassName( timerObjectNode, countdownElem );
```

Reply:
Went for this instead